server/data-import/data.js

```diff
@@ -25,23 +22,36 @@ loadData()
 
 // note that exports are 'mongoexport --db hah-dev --collection pregnancycenters --jsonArray --out cessilye_nypc_geocoded.json'
 
-PregnancyCenterModel.collection.dropAsync()
-	.catch( (err) => log.error(err))
-	.then( () => {
-		fs.readFile('test/fixtures/cessilye_nypc.json', 'utf8', (err, data) => {
-			const docs = EJSON.parse(data)
-			log.info(typeof docs)
-			for (const doc of docs) {
+async function reimport() {
 
-				Joi.validateAsync(doc, pregnancyCenterSchemaJoi, {abortEarly: false})
-					.catch( (err) => log.error(err))
-					.then( (validatedData) => {
-						PregnancyCenterModel.createAsync(validatedData)
-							.then( (pc) => log.info(pc))
-					})
+	let docs
 
-			}
+	try {
+		await PregnancyCenterModel.collection.drop()
+	} catch (err) {
+		log.error(err)
+	}
 
-		})
+	try {
+		const dataToImport = fs.readFileSync('server/data-import/test/fixtures/cessilye_nypc.json', 'utf8')
+		docs = EJSON.parse(dataToImport)
+	} catch (err) {
+		log.error(err)
+	}
 
-	})
+	for (const doc of docs) {
+		const validationObj = await Joi.validate(doc, pregnancyCenterSchemaJoi, {
+			abortEarly: false
+		})
+		if (validationObj.error) {
+			log.error(validationObj.error)
```

Comment:
If it fails, the Joi object has an object under the 'error' attribute that gives a (relatively) user-friendly description of the problem. The tests have some good examples of what it looks like. For example, if the phone number isn't formatted correctly, the error is 'child "phone" fails because ["phone" needs to be a valid phone number according to E.164 international format]' 

It's an asynchronous call. Without this, I'd need to use callbacks or promises, so this looks a lot cleaner.